### ISSUE TITLE
@PathParam on interfaces

### ISSUE THREAD

**Issue by ssoltanid** [swagger-api/swagger-core#1584]
Doesn't recognize the Long param and show a String body param.

Interface : 

```java
    public interface RepositoryService<T extends AbstractModel> {
    @Path("/{idEntity}")
    @DELETE
    @ApiOperation(value = "Logical delete an entity",
            notes = "Must provide an id for the entity to delete")
    Response deleteEntity(@PathParam("idEntity") Long idEntity);
    }
```
Implementation : 

```java
    
    @Path("catalogueService")
    @Api(tags = "Catalogue Service WS")
    public class CatalogueServiceImpl implements RepositoryService{

    @Override
    public Response deleteEntity(Long idEntity) {
       ....
    }
    }
```

Any ideas ?

**Comment by fehguy** [comment]
Best thing you can do is create a test case in the same structure as in swagger-core and we can tackle it.

**Comment by janhoeve** [comment]
Seems to be a duplicate of #1506

**Comment by frantuma** [comment]
should be solved in #1747
